app/src/main/java/ru/netology/fmhandroid/utils/Utils.kt

```diff
@@ -0,0 +1,166 @@
+package ru.netology.fmhandroid.utils
+
+import android.content.Context
+import android.net.ConnectivityManager
+import android.net.NetworkCapabilities
+import android.os.Build
+import android.widget.EditText
+import retrofit2.Response
+import ru.netology.fmhandroid.dto.ClaimComment
+import ru.netology.fmhandroid.exceptions.ApiException
+import ru.netology.fmhandroid.exceptions.ServerException
+import ru.netology.fmhandroid.exceptions.UnknownException
+import java.io.IOException
+import java.text.SimpleDateFormat
+import java.time.*
+import java.time.format.DateTimeFormatter
+import java.util.*
+
+
+object Utils {
+
+    object EmptyComment {
+        val emptyClaimComment = ClaimComment(
+            id = 0,
+            claimId = 0,
+            description = "",
+            creatorId = 0,
+            createDate = 0,
+        )
+    }
+
+    fun convertNewsCategory(category: String): Int {
+        return when (category) {
+            "Объявление" -> 1
+            "День рождения" -> 2
+            "Зарплата" -> 3
+            "Профсоюз" -> 4
+            "Праздник" -> 5
+            "Массаж" -> 6
+            "Благодарность" -> 7
+            "Нужна помощь" -> 8
+            else -> 0
+        }
+    }
+
+    fun updateDateLabel(calendar: Calendar, editText: EditText) {
+        val format = "dd.MM.yyyy"
+        val simpleDateFormat = SimpleDateFormat(format, Locale.getDefault())
+        editText.setText(simpleDateFormat.format(calendar.time))
+    }

```

Comment:
А если вынести из `Utils` и превратить в расширение для `EditText`?

```suggestion
internal fun EditText.setDate(calendar: Calendar) {
    val format = "dd.MM.yyyy"
    val simpleDateFormat = SimpleDateFormat(format, Locale.getDefault())
    setText(simpleDateFormat.format(calendar.time))
}
```